scripts/speedtest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/home/pi/.local/bin/speedtest-cli --json > /home/pi/PiWeatherRock/speedtest/queue/$(date +%s).json
```

Comment:
where is `speedtest-cli` coming from? Also, this is another place where I want to avoid hardcoded paths in the provided files. Assuming the file is indeed needed, I think we can create it using a template and Puppet.

Reply:
Added `speedtest-cli` to scripts dir, along with it's license file. Removed hard-coded paths, too. I emailed the dev about proper attribution and how best to included the `Apache 2.0` license file

Reply:
I originally forgot how I got `speedtest-cli` on the Pi, but it looks like it was a pip install. I don't think we need to include the LICENSE file if it's installed through pip, but I'm not sure.